.github/workflows/build_docs.yaml

```diff
@@ -18,31 +18,37 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.11'
+          python-version: "3.11"
           cache: "pip"
           cache-dependency-path: "setup.cfg"
 
-      - name: Install graphviz
-        run: sudo apt-get install graphviz
-
-      - name: Install sqlite
-        run: sudo apt-get install sqlite libyaml-dev
+      - name: Install uv
+        run: |
+          python -m pip install --upgrade pip
+          pip install uv
 
-      - name: Update pip/wheel infrastructure
+      - name: Update wheel infrastructure
         run: |
-          python -m pip install --upgrade pip uv
           uv pip install --system wheel
 
+      - name: Install postgresql (server)
+        run: |
+          sudo apt-get update
+          sudo apt-get install postgresql
+
       - name: Install dependencies
         run: |
           uv pip install --system -r requirements.txt
 
       - name: Build and install
-        run: uv pip install --system --no-deps -v .
+        run: uv pip install --system --no-deps -v -e .
+
+      - name: Install graphviz
+        run: sudo apt-get install graphviz
 
       - name: Install documenteer
         run: uv pip install --system 'documenteer[pipelines]==0.8.2'
 
       - name: Build documentation
         working-directory: ./doc
-        run: package-docs build -W
+        run: package-docs build
```

Comment:
Why are you removing the warnings?

Reply:
I blindly copied what is in daf_butler.

Reply:
What was wrong with the action before you did this then? I thought it was working fine.

Reply:
I think removing the sqlite was going to be the fix.

Reply:
Restored warnings, and detected a duplicate definition in the RST. Thanks, Tim!